config/cmake/modules/MfemCmakeUtilities.cmake

```diff
@@ -157,6 +150,21 @@ macro(add_mfem_miniapp MFEM_EXE_NAME)
   endif()
 endmacro()
 
+# Macro for setting variables like '<culib>_LIBRARIES' where <culib> is a CUDA
+# library like cublas. This macro assumes that the CUDAToolkit module was loaded
+# successfully. Example usage:
+#   mfem_culib_set_libraries(CUBLAS cublas)
+macro(mfem_culib_set_libraries _CULIB _culib)
+  # The following command do not work with older CMake versions, e.g. 3.20:
```

Comment:
does not

Reply:
3.20 confused me. CMake minimal version 3.12 is required.

Reply:
Typo fixed.

I'm not sure what is confusing about the statement: CMake 3.20 is a supported older version of CMake. If you have a suggestion how to improve it, let me know.